bindata/network/ovn-kubernetes/managed/004-config.yaml

```diff
@@ -147,12 +141,6 @@ data:
     [gateway]
     mode={{.OVN_GATEWAY_MODE}}
     nodeport=true
-{{- if (index . "V4InternalMasqueradeSubnet")}}
```

Comment:
Prior to this change `V[4|6]InternalMasqueradeSubnet` was set in the configmap so it was read by both ovnkube-node daemonset and the cluster-manager deployment. With your PR only ovnkube-node gets the argument.
I think it would be good to add it to control-plane too, for consistency, even if it is not currently used.

transit cidr example:
https://github.com/openshift/cluster-network-operator/blob/0be5bd93a6cba203b77313b3444022330a61b58a/bindata/network/ovn-kubernetes/self-hosted/ovnkube-control-plane.yaml#L118-L125
https://github.com/openshift/cluster-network-operator/blob/4776726dee1d22b6b06bbaa7024fc0d3776b6f97/bindata/network/ovn-kubernetes/managed/ovnkube-control-plane.yaml#L158-L165

Reply:
@kyrtapz  I omitted it as we dont need masquerade subnet while running cluster-manager. Similarly we dont provide transit switch subnet details to ovnkube-controller. 
However if its about consistency then I agree that adding masquerade subnet to ovnkube-control-plane.yaml makes sense.

Reply:
Thanks! so currently:
- JoinSubnet - passed to both
- TransitSwitchSubnet - only passed to cluster-manager
- MasqueradeSubnet - only passed to ovnkube-node

Prior to your change MasqueradeSubnet was configured through the configmap for both, I am ok to keep it as is for now if we were not using the  MasqueradeSubnet in cluster-manager anyway.